src/storage/src/hummock/local_version/pinned_version.rs

```diff
@@ -126,15 +134,26 @@ impl PinnedVersion {
         self.version.id != INVALID_VERSION_ID
     }
 
-    pub fn levels(&self, new_compaction_group_id: CompactionGroupId) -> Vec<&Level> {
+    pub fn levels_by_cg(&self, compaction_group_id: CompactionGroupId) -> Vec<&Level> {
```

Comment:
`levels_by_compaction_group_id` is more easy to read for me

Reply:
> `levels_by_compaction_group_id` is more easy to read for me

fixed